daprdocs/content/en/operations/support/support-release-policy.md

```diff
@@ -95,9 +95,14 @@ After announcing a future breaking change, the change will happen in 2 releases
 ## Upgrade on Hosting platforms
 Dapr can support multiple hosting platforms for production. With the 1.0 release the two supported platforms are Kubernetes and physical machines. For Kubernetes upgrades see [Production guidelines on Kubernetes]({{< ref kubernetes-production.md >}})
 
-### Supported Kubernetes versions
-
-Dapr follows [Kubernetes Version Skew Policy](https://kubernetes.io/releases/version-skew-policy).
+### Supported versions Dapr versions
+Below is a list of software that the latest version of Dapr (v{{% dapr-latest-version long="true" %}}) has been tested against. 
+
+| Dependency            |   Supported Version                                                                                                              |
+|-----------------------|----------------------------------------------------------------------------------------------------------------------------------|
+| Kubernetes                                                | Dapr support for Kubernetes is aligned with [Kubernetes Version Skew Policy](https://kubernetes.io/releases/version-skew-policy/)|
+| OTEL                                                     |                                                                                                                            v0.4.0|
```

Comment:
Use the full name Open Telemetry (OTEL) , since not everyone will know this acronym.
Also how about a link to https://github.com/open-telemetry? Same as you have done for Prometheus

Reply:
Done